src/XmlUtils.cc

```diff
@@ -58,10 +74,21 @@ tinyxml2::XMLNode *DeepClone(tinyxml2::XMLDocument *_doc,
 
 /////////////////////////////////////////////////
 std::string ElementToString(const tinyxml2::XMLElement *_elem)
+{
+  sdf::Errors errors;
+  std::string result = ElementToString(_elem, errors);
+  sdf::throwOrPrintErrors(errors);
+  return result;
+}
+
+/////////////////////////////////////////////////
+std::string ElementToString(const tinyxml2::XMLElement *_elem,
```

Comment:
`sdf::Errors` should be the first argument to be consistent with the rest of the API.

Reply:
https://github.com/gazebosim/sdformat/pull/1296/commits/c41ec9cf8ce88056773244a3ef7aa2df9fa02955